sympy/printing/fcode.py

```diff
@@ -103,7 +104,9 @@ class FCodePrinter(CodePrinter):
         '!=': '/=',
     }
 
-    def __init__(self, settings={}):
+    def __init__(self, settings={}, mangled_symbols={}):
+        self.mangled_symbols = mangled_symbols         ## Dict showing mapping of all words
```

Comment:
This needs to read `self.mangled_symbols = {}`. (drop `mangled_symbols` as keyword argument)